apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/api/ApexGrammar.java

```diff
@@ -28,23 +29,26 @@ private ApexGrammar() {
      * It is the main method of grammar. Here all other grammars are
      * constructed.
      *
+     * @param errorRecoveryEnabled error recovery status
      * @return the grammar
      */
-    public static Grammar create() {
+    public static Grammar create(boolean errorRecoveryEnabled) {
```

Comment:
Yes please, we should keep it as simple as possible, if at some moment we need something similar we will think about it.